### PR TITLE
Create [Term Entry] SQL Math Functions EXP

### PR DESCRIPTION
**Pull request by andyguo0228**
### Description

- Added SQL math function for `EXP()`
- Closes issue #2258 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/math-functions/terms/exp/exp.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - 'SQL Server'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'Math'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+**`EXP()`** is a mathematical function in SQL that returns the value of **e** raised to the power of a given `number`. Where **e** is the base of natural logarithms, a mathematical constant equal to approximately 2.71828, and `number` is the exponent e is raised to.
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/exp/exp.md:18]:
```suggestion
**`EXP()`** is a mathematical function in SQL that returns the value of _e_ raised to the power of a given number. Where _e_ is the base of natural logarithms, a mathematical constant equal to approximately 2.71828.
```

File: content/sql/concepts/math-functions/terms/exp/exp.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Syntax
+
+```sql
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/exp/exp.md:22]: This should be a ` ```pseudo ` code block, and should show the function outside any `SELECT` statements, `SELECT` is not a part of the syntax of `EXP()`

Also follow the code block with a sentence describing what `number` is in the example.

File: content/sql/concepts/math-functions/terms/exp/exp.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+The following table `exp` contains example data:
+
+| id | num |
+|----|-----|
+| 1  | 0   |
+| 2  | 1   |
+| 3  | 2   |
+
+The `EXP()` function is used to return the value of e raised to the power of the `num` column:
```
Comment (SSwiniarski) [inline content/sql/concepts/math-functions/terms/exp/exp.md:37]:
```suggestion
The `EXP()` function is used to return the value of _e_ raised to the power of the `num` column:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@andyguo0228 I did a first review and left a few comments.

**Comment by andyguo0228** [comment]
@SSwiniarski I applied the recommended changes.

**Review by SSwiniarski — APPROVED** [review]

File: content/sql/concepts/math-functions/terms/exp/exp.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  - 'SQL Server'
+  - 'PostgreSQL'
+  - 'MySQL'
+  - 'Math'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+**`EXP()`** is a mathematical function in SQL that returns the value of _e_ raised to the power of a given number. Where _e_ is the base of natural logarithms, a mathematical constant equal to approximately 2.71828.
```
Comment (caupolicandiaz) [inline content/sql/concepts/math-functions/terms/exp/exp.md:18]:
```suggestion
**`EXP()`** is a mathematical function in SQL that returns the value of _e_ raised to the power of a given number. Where _e_ is a mathematical constant equal to approximately 2.71828, which is the base used in the determination of natural logarithms.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@andyguo0228 Just one comment. Please update the entry, and tag me when you've made the changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@andyguo0228 @SSwiniarski LGTM

**Comment by SSwiniarski** [comment]
@andyguo0228 Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/sql/math-functions/exp

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**